nx/include/switch/services/mii.h

```diff
@@ -0,0 +1,176 @@
+/**
+ * @file mii.h
+ * @brief Mii services (mii:*) IPC wrapper.
+ * @author XorTroll
+ * @copyright libnx Authors
+ */
+#pragma once
+#include "../types.h"
+#include "../sf/service.h"
+
+typedef enum {
+    MiiServiceType_System  = 0,  ///< Initializes mii:e.
+    MiiServiceType_User    = 1,  ///< Initializes mii:u.
+} MiiServiceType;
+
+/// Mii age.
+typedef enum {
+    MiiAge_Young   = 0,  ///< Young
+    MiiAge_Normal  = 1,  ///< Normal
+    MiiAge_Old     = 2,  ///< Old
+    MiiAge_All     = 3,  ///< All of them
+} MiiAge;
+
+/// Mii gender.
+typedef enum {
+    MiiGender_Male    = 0,  ///< Male
+    MiiGender_Female  = 1,  ///< Female
+    MiiGender_All     = 2,  ///< Both of them
+} MiiGender;
+
+/// Mii race.
```

Comment:
"Race" is a loaded term which should probably be avoided. I suggest `MiiFaceColor` (which seems to be used in other places). Also, I'm interested in knowing if there is an official name for this enum and its values.

Reply:
Agree, will rename it, but according to RE and Ryujinx's implementation this is the official name

Reply:
Also my bad, the official name seems to be FaceColor (it's used in the sdk, at least), just that since Ryujinx had it named as Race I just copied that. Renamed it fo FaceColor

Reply:
Going by the `All` field this enum is probably separate from the normal color enums?(same with the other enums?)

Reply:
The All fields are used for the BuildRandom command in order to make a completely random mii, that's why those three enums have it